### ISSUE TITLE
Set focus on card's first widget when card loads in editor or workflow step

### ISSUE THREAD

**Issue by chiatt** [archesproject/arches#8167]
To ease data entry, when a card loads in the editor or in a workflow, the browser focus should be applied to the card's first widget. This could be accomplished by identifying the widget index in the card, and then passing the index to the widget. If the widget is the first widget, the knockout 'hasFocus' binding can be used to set the focus. Each widget will need to be updated to implement this, as each may set focus differently.

**Comment by chiatt** [comment]
@phudson-he would this change have any implications for accessibility?

**Comment by phudson-he** [comment]
@chiatt Yes it would aid the user, especially a keyboard only user, the focus should change to the first tabbable item in the card that has just loaded (logical/meaningful order). Covered by [WCAG 2.1 - 2.4.3](https://www.w3.org/TR/WCAG21/#focus-order). This is particularly important for any popups of modals.

**Comment by chiatt** [comment]
That's great - thanks @phudson-he
